### ISSUE TITLE
 arubaoss_command send executed command to stdout

### ISSUE THREAD

**Issue by NeverUsedID** [aruba/aos-switch-ansible-collection#13]
Example task:  
    - name: Get Firmware
      arubaoss_command:
        commands:
          -  sh version | i WC
      register: get_firmware

Debug output:

  "get_firmware": {
        "changed": false,
        "failed": false,
        "stdout": [
            "**sh version | i WC**                WC.16.10.0015\nBoot ROM Version:    WC.16.01.0008"
        ],
        "stdout_lines": [
            [
                "**sh version | i WC**                WC.16.10.0015",
                "Boot ROM Version:    WC.16.01.0008"
            ]
        ]
    }

 How do i remove the entered command from stdout?

**Comment by tchiapuziowong** [comment]
Hi @NeverUsedID !

You'll want to use Ansible's `replace` filter in order to remove your command from the output. See Ansible's documentation here on how to use regular expressions - the third example explains `replace` - hope this helps!

https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#searching-strings-with-regular-expressions

**Comment by NeverUsedID** [comment]
Yes, I'm using this, this way at the moment, i just was surprised, as i havnt seen this practice in any other module for ansible, and thought that there is an option to diable it.
